packages/app/src/systems/Home/pages/Home/Home.tsx

```diff
@@ -21,16 +23,21 @@ export function Home() {
     navigate(Pages.receive());
   }, [navigate]);
 
+  const goToAccounts = useCallback(() => {
```

Comment:
same here, i think you don't need a `useCallback` here... `useCallback` is better used when you have heavy computations using the component scope and want to avoid re-create these computations in all re-renders you can use it, but for simple cases like that you don't need... in fact, is very rare when you, in fact, will need it

Reply:
I agree I used here just to keep consistency on the code.